packages/web/src/components/gcds-heading/stories/gcds-heading.stories.tsx

```diff
@@ -0,0 +1,190 @@
+export default {
+  title: 'Components/Heading',
+
+  argTypes: {
+    // Props
+    tag: {
+      control: { type: 'select' },
+      options: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'],
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+      type: {
+        required: true,
+      },
+    },
+    characterLimit: {
+      name: 'character-limit',
+      control: 'boolean',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: true },
+      },
+    },
+    marginTop: {
+      name: 'margin-top',
+      control: { type: 'select' },
+      options: [
+        '0',
+        '50',
+        '100',
+        '150',
+        '200',
+        '250',
+        '300',
+        '400',
+        '450',
+        '500',
+        '550',
+        '600',
+        '700',
+        '800',
+        '900',
+        '1000',
+      ],
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'h1: 0, h2-h6: 500' },
```

Comment:
Does this look ok as default value or is it weird to display the different values?

Reply:
I really like this solution!

Are we going to also display the default value for the margin-bottom in storybook?

Reply:
We sure are - good catch! I just added it in.